azure-pipelines.yml

```diff
@@ -94,6 +94,63 @@ stages:
 
   #######################################
 
+  - job: RPIBuilds
+    pool:
+      name: Default
+      demands:
+        - agent.name -equals iio-osc_rpi
+    workspace:
+      clean: all
+    steps:
+    - checkout: self
+      fetchDepth: 1
+      clean: true
+      persistCredentials: true
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        source: 'specific'
+        project: '$(System.TeamProjectId)'
+        pipeline: $(libiioPipelineId)
+        artifact: 'Ubuntu-arm32v7'
+        runVersion: 'latestFromBranch'
+        runBranch: 'refs/heads/master'
+        path: '$(Agent.BuildDirectory)/s/download/'
+      displayName: 'Get libiio artifacts'
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        source: 'specific'
+        project: '$(System.TeamProjectId)'
+        pipeline: $(libad9361iioPipelineId)
+        artifact: 'Ubuntu-arm32v7'
+        runVersion: 'latestFromBranch'
+        runBranch: 'refs/heads/master'
+        path: '$(Agent.BuildDirectory)/s/download/'
+      displayName: 'Get libad9361-iio artifacts'
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        source: 'specific'
+        project: '$(System.TeamProjectId)'
+        pipeline: $(libad9166iioPipelineId)
+        artifact: 'Ubuntu-arm32v7'
+        runVersion: 'latestFromBranch'
+        runBranch: 'refs/heads/master'
+        path: '$(Agent.BuildDirectory)/s/download/'
+      displayName: 'Get libad9166-iio artifacts'
+    - script: |
+        sudo dpkg -i ./download/*.deb
+      displayName: 'Install Dependencies'
+    - script: |
+        mkdir build && cd build
+        cmake ../
+        make -j9
```

Comment:
RPI do not have 9 CPU cores. This kind of builds will just kill it (freeze it) for at least few minutes. 
Please check how many cores its CPU has and set make command to use maximum <total number of cores minus one> threads.